spring-cloud-dataflow-admin-local/src/main/resources/admin.yml

```diff
@@ -2,7 +2,11 @@ server:
   port: 9393
 management:
   contextPath: /management
-
+info:
```

Comment:
Is there a way to define these `info.app.*` properties via admin-starter so that we don't have to explicitly specify these properties for each SPI implementation?